ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -249,13 +256,34 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         private logger: LoggingService,
         private ngxTranslate: NGXTranslateService,
         @Inject('ddp.config') private configuration: ConfigurationService,
-        @Optional() private submitService: SubmitAnnouncementService
+        @Optional() private submitService: SubmitAnnouncementService,
+        private windowRef: WindowRef
     ) {
         this.suggestionForm = new FormGroup({
             suggestionRadioGroup: new FormControl('entered')
         });
         this.initializeComponentState();
     }
+    
+    private setupScrollToErrorAction(): void {
```

Comment:
this method was needed to be added since it handles scroll-up functionality in the case of 'addressEmbedded' component.

Reply:
That makes sense for me. 
After the fix, we will have same scroll up to a failed validation field feature for Mailing address as well.